src/DataFlowTasks.jl

```diff
@@ -35,7 +35,7 @@ include("dataflowtask.jl")
 include("arrayinterface.jl")
 include("dag.jl")
 include("scheduler.jl")
-include("otherschedulers.jl")
+# include("otherschedulers.jl")
```

Comment:
I am ignoring the `otherschedulers.jl` file for now, as well as its tests. If we decide that we have no plan to support other scheduler types, then lots of the code in the package can be significantly simplified. Something to think about...